benchexec/tools/tabol.py

```diff
@@ -0,0 +1,47 @@
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+Copyright (C) 2007-2015  Dirk Beyer
+All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import benchexec.result as result
+import benchexec.util as util
+import benchexec.tools.template
+
+class Tool(benchexec.tools.template.BaseTool):
+
+    def executable(self):
+        return util.find_executable('tabol.jar')
+
+    def name(self):
+        return 'TABOL'
+
+    def cmdline(self, executable, options, tasks, propertyfile, rlimits):
+		
+        return "java -jar " + [executable] + tasks
+
+    def determine_result(self, returncode, returnsignal, output, isTimeout):
+
+        status = result.RESULT_UNKNOWN
+        stroutput = str(output)
```

Comment:
`output` is a list of lines, so `str(output)` will give something like `['line1', 'line2']`. This will in principle work, but might be unexpected during future changes. Something like `"\n".join(output)` might be more intuitive (or just loop over the lines).